torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -622,6 +623,34 @@ Tensor _sparse_addmm_sparse_backward(const Tensor& grad, const Tensor& sparse_,
   return grad_sparse.sparse_mask(sparse);
 }
 
+Tensor _sparse_matrix_mask(const Tensor& grad, const Tensor& mask){
+  Tensor output = at::native::empty_like(mask);
+  if (mask._nnz() == 0) {
+    return output.zero_();
+  }
+  Tensor mask_indices = at::empty_like(mask._indices());
+  mask_indices.copy_(mask._indices());
+  auto r_values = _sparse_matrix_mask_helper(mask._nnz(), grad, mask_indices.contiguous());
```

Comment:
nit: you don't need `mask._nnz()` argument, you can compute in inside the function as `mask_indices.size()[1]`